crypto/aes/aes_ige.c

```diff
@@ -38,15 +39,31 @@ void AES_ige_encrypt(const unsigned char *in, unsigned char *out,
                      size_t length, const AES_KEY *key,
                      unsigned char *ivec, const int enc)
 {
+    /* Ignore the return value since we are void */
+    int ret =AES_ige_encrypt_ex(in, out, length, key, ivec, enc);
+
+    assert(ret);
```

Comment:
Yes - its a problem if I put the call to AES_ige_encrypt_ex() inside the assert as suggested...because then AES_ige_encrypt_ex() doesn't get called at all, and the function does nothing!